pyrightconfig.json

```diff
@@ -1,4 +1,5 @@
 {
-    "include": ["./elements/pl-order-blocks"],
-    "extraPaths": ["./python"]
+    "include": ["./elements/pl-hidden-hints"],
+    "extraPaths": ["./python"],
+    "pythonVersion": "3.8"
```

Comment:
This is what's on the live servers (as far as I know). We should double check with Nate.

Reply:
Yep, we're still at a minimum of 3.8 right now.

Reply:
Declaring the python version explicitly is actually _super_ important. Without this, pyright could default to 3.10 (or some other more advanced version), and then typecheck allowing 3.10 syntax. If PL servers are on 3.8, pyright won't throw an error if some non-backwards-compatible 3.10 syntax is used in typechecked files.

Reply:
That may be true of someone's local dev environment, but it would never reach production. During the automated CI testing that is done on all pull requests, the tests will be run in a docker container that has the right version for production.

Reply:
huh, actually I was just poking around the `plbase` docker container and it seems the python version isn't declared there anywhere that I can find, so I'm not sure how the version of python used for the CI testing is decided. 

In any case, if we are going to pin a particular version of python for CI tests, it should be done in one place, not in multiple places, and it seems like doing that in the container creation makes more sense than doing it in the type checker config. @nwalters512 do you have thoughts on this?

Reply:
Also, last I checked with @mwest1066 there was still maybe one or two servers running something lower, like 3.7. Is that still true?

Reply:
It looks like the CI version is running python 3.10 though? (see screenshot) I'm not even sure how much the installed python version matters for pyright since it uses node.

![Screenshot_20221003_125825](https://user-images.githubusercontent.com/1345068/193646302-1b2cc8bc-346d-4f61-90da-9d6ce17e31ae.png)

Reply:
Ok, I just dug up some old messages and realized that I misremembered which version Matt had said before. 

So, I guess fixing this at 3.8 should be fine. 

On the other hand, it still seems problematic to me that our CI is running on 3.10 when we have servers on 3.8 for reasons aside from just type checking. That is probably something we should fix, but that's outside of the scope of this PR.